packages/components/src/number-control/index.js

```diff
@@ -13,13 +12,14 @@ import { forwardRef } from '@wordpress/element';
  * Internal dependencies
  */
 import { Input } from './styles/number-control-styles';
-import { add, getValue, roundClamp, subtract } from './utils';
-import { isValueEmpty } from '../input-control/utils';
 import {
 	inputControlActionTypes,
 	composeStateReducers,
 } from '../input-control/state';
 import { useRTL } from '../utils/style-mixins';
+import { add, subtract, roundClamp } from '../utils/math';
```

Comment:
Abstracted various functions it new dedicated `utils/*.js` files, which are shared amongst various components.